crates/bevy_render/src/view/window/cursor.rs

```diff
@@ -29,7 +28,7 @@ impl Plugin for CursorPlugin {
         app.register_type::<CursorIcon>()
             .init_resource::<CustomCursorCache>()
             .add_systems(Last, update_cursors)
-            .add_systems(Update, add_cursor_to_windows);
+            .observe(add_cursor_to_windows);
```

Comment:
@eero-lehtinen this was quite pretty to do, I remember reading the example for this last week and now this seems so nice